jetty/src/test/scala/org/http4s/server/jetty/Issue454.scala

```diff
@@ -18,15 +18,16 @@ package org.http4s
 package server
 package jetty
 
-import cats.effect.{ContextShift, IO}
+import cats.effect.IO
+import cats.effect.std.Dispatcher
 import org.eclipse.jetty.server.{HttpConfiguration, HttpConnectionFactory, Server, ServerConnector}
 import org.eclipse.jetty.servlet.{ServletContextHandler, ServletHolder}
 import org.http4s.dsl.io._
 import org.http4s.servlet.AsyncHttp4sServlet
 import org.http4s.syntax.all._
 
 object Issue454 {
-  implicit val cs: ContextShift[IO] = Http4sSpec.TestContextShift
+  // implicit val cs: ContextShift[IO] = Http4sSpec.TestContextShift
```

Comment:
This code doesn't ever run as part of the build.  If we can't figure out a way to automate it, then I think it's ready to be removed.

Reply:
removed it